base_layer/common_types/src/tari_address/mod.rs

```diff
@@ -48,15 +47,18 @@ const INTERNAL_DUAL_BASE58_MIN_SIZE: usize = 89; // number of bytes used for the
 const INTERNAL_DUAL_BASE58_MAX_SIZE: usize = 91; // number of bytes used for the internal representation
 const INTERNAL_SINGLE_MIN_BASE58_SIZE: usize = 45; // number of bytes used for the internal representation
 const INTERNAL_SINGLE_MAX_BASE58_SIZE: usize = 48; // number of bytes used for the internal representation
+const MAX_ENCRYPTED_DATA_SIZE: usize = 256; // max size of the payment_id_ bytes
```

Comment:
We should reference a const from `encrypted_data.rs` here

Reply:
cant its going to break the dependancy chain.